jax/lib/xla_bridge.py

```diff
@@ -34,6 +34,7 @@
 from .. import dtypes
 import numpy as onp  # 'onp' rather than 'np' to distinguish from autograd.numpy
 import threading
+import jax.numpy as np
```

Comment:
We also don't want to import `jax.numpy` here. In general, be careful with adding new import statements, since dependencies can be tricky to get right!